### PR TITLE
chore: install buildkite test analytics (experiment)

### PR DESCRIPTION
**Pull request by millsp**
https://buildkite.com/test-analytics

### CODE REVIEW

**Comment by janpio** [comment]
😢 
image

File: .github/workflows/scripts/upload-buildkite-junit.sh [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  -H "Authorization: Token token=\"$BUILDKITE_ANALYTICS_TOKEN\"" \
+  -F "data=@$1" \
+  -F "format=junit" \
+  -F "run_env[CI]=github_actions" \
+  -F "run_env[key]=$GITHUB_ACTION-$GITHUB_RUN_NUMBER-$GITHUB_RUN_ATTEMPT" \
+  -F "run_env[number]=$GITHUB_RUN_NUMBER" \
+  -F "run_env[branch]=$GITHUB_REF" \
+  -F "run_env[commit_sha]=$GITHUB_SHA" \
+  -F "run_env[url]=https://github.com/$GITHUB_REPOSITORY/actions/runs/$GITHUB_RUN_ID" \
+  https://analytics-api.buildkite.com/v1/uploads
```
Comment (Jolg42) [inline .github/workflows/scripts/upload-buildkite-junit.sh:16]: Let's see how this goes, possible we need to add a retry

File: packages/react-prisma/jest.config.js [hunk 2]
```diff
@@ -17,5 +17,7 @@ module.exports = {
         titleTemplate: '{title}',
       },
     ],
+    'buildkite-test-collector/jest/reporter',
   ],
+  testLocationInResults: true,
```
Comment (Jolg42) [inline packages/react-prisma/jest.config.js:22]: There is no test running here, not sure we need it?

File: .github/workflows/test.yml [hunk 3]
```diff
@@ -179,6 +179,18 @@ jobs:
           flags: client,${{ matrix.os }},${{ matrix.queryEngine }}
           name: client-${{ matrix.os }}-${{ matrix.queryEngine }}
 
+      - run: bash .github/workflows/scripts/upload-buildkite-junit.sh packages/client/junit.xml
```
Comment (Jolg42) [inline .github/workflows/test.yml:182]:
```suggestion
      - name: Upload test results to Buildkite test analytics
      	run: bash .github/workflows/scripts/upload-buildkite-junit.sh packages/client/junit.xml
```

**Review by Jolg42 — APPROVED** [review]

**Pull request closed without merging**